app/controllers/users_controller.rb

```diff
@@ -88,7 +88,10 @@ def destroy
   end
 
   def bulk_destroy
-    @users = load_selected_objects(User)
+    @users = User.accessible_by(current_ability)
```

Comment:
Same here. Although no easy way to test this one. So just specify :destroy here and that's good enough.